config/secrets.js

```diff
@@ -38,6 +38,14 @@ module.exports = {
     clientID: process.env.LINKEDIN_ID,
     clientSecret: process.env.LINKEDIN_SECRET,
     callbackURL: '/auth/linkedin/callback',
+    profileFields: [
```

Comment:
These fields need a review.

Reply:
I think, only the profile-url is needed for our purposes, and only adding that should be fine.